src/game_engine/unreal/mod.rs

```diff
@@ -217,7 +220,7 @@ impl UClass {
         &'a self,
         process: &'a Process,
         module: &'a Module,
-    ) -> impl FusedIterator<Item = UProperty> + '_ {
+    ) -> impl FusedIterator<Item = UProperty> + 'a {
```

Comment:
Why is this returning 'a?

Reply:
uhhh it really shouldn't. Probably leftover data or clippy lints that stayed behind. Weird, I admit lol

Reply:
Oh ok I know now. Clippy complains because the lifetime is known to be 'a